shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -746,6 +748,21 @@ - (void)testHideOverlay {
   engine.viewController = nil;
 }
 
+- (void)testHideA11yElements {
+  FlutterDartProject* project = [[FlutterDartProject alloc] init];
+  FlutterEngine* engine = [[FlutterEngine alloc] initWithName:@"foobar" project:project];
+  [engine createShell:@"" libraryURI:@"" initialRoute:nil];
+  FlutterViewController* realVC = [[FlutterViewController alloc] initWithEngine:engine
+                                                                        nibName:nil
+                                                                         bundle:nil];
+  XCTAssertFalse(realVC.view.accessibilityElementsHidden, @"");
```

Comment:
Posting the notifications directly instead of exposing the `applicationWillResignActive:` `applicationBecameActive:` API in the test worked for me:
```objc
  XCTAssertFalse(realVC.view.accessibilityElementsHidden);
  [[NSNotificationCenter defaultCenter] postNotificationName:UIApplicationWillResignActiveNotification object:nil];
  XCTAssertTrue(realVC.view.accessibilityElementsHidden);
  [[NSNotificationCenter defaultCenter] postNotificationName:UIApplicationDidBecomeActiveNotification object:nil];
  XCTAssertFalse(realVC.view.accessibilityElementsHidden);
```

Reply:
I went with this. I think I was somehow not rebuilding the target correctly when I got failures on this locally. It's not failing now.